revamp-service/src/main/resources/application.properties

```diff
@@ -22,4 +22,4 @@ logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
 spring.jackson.time-zone= America/New_York
 metadata-complete="true"
 #app related properites
-image.path=C:\\temp\\Puthuyir_Images
+image.path=D:\\workspace_putholi\\putholi_images
```

Comment:
Also is it possible to have the same dir structure: "C:\\temp\\Puthuyir_Images"?